### DISCUSSION TITLE
Wont Change palettes!

### DISCUSSION THREAD

**Discussion by Idkbythispoint** [BEEmod/BEE2-items#4202] [Tech Support / Q&A] [answered]
Please help me my palettes wont update in portal 2!

**Comment by Idkbythispoint** [comment]
Please help me.

**Comment by ghost** [comment]
Are you exporting it?

**Comment by Idkbythispoint** [comment]
no

**Comment by Idkbythispoint** [comment]
im making it but when i open portal 2 it stays how it is if you dont have beemod and you have a non-modded portal 2

**Comment by Idkbythispoint** [comment]
hello?

**Comment by ghost** [comment]
Are you exporting the palette, if not then that's why you are not getting your wanted items

**Comment by ghost** [comment]
If not, go to file then export.

**Comment by Idkbythispoint** [comment]
like what do i do?

**Comment by ghost** [comment, marked as answer]
Top left corner of the program, you should see File, click on that then find Export and click it, wait then play the game

**Comment by Idkbythispoint** [comment]
Thanks, it worked.

**Comment by Idkbythispoint** [comment]
also how do i make a new  palette?

**Comment by Idkbythispoint** [comment]
nvm this case is closed now because i have fugured it out and got help, Bye and Thanks!
